descriptors/ModuleDescriptor-template.json

```diff
@@ -252,6 +252,14 @@
       "description": "Removes feature configuration"
     }
   ],
+  "user": {
```

Comment:
I don't think this is legal within the module descriptor schema, which has `"additionalProperties": false`. It might be able to be embedded within the `metadata` property of the module descriptor. Have you tried posting this module descriptor to Okapi? @adamdickmeiss may have more insight.

Reply:
Right. Not legal

Reply:
@wafschneider this is addressed in https://github.com/folio-org/okapi/pull/1168